app/Http/Controllers/Front/ClientController.php

```diff
@@ -260,7 +260,7 @@ public function getInvoices()
                             }
                         })
                     ->addColumn('date', function ($model) {
-                        return getDateHtml($model->date);
+                        return getDateHtmlcopy($model->created_at);
                     })
```

Comment:
Do not change this from date to created_at. because In the admin panel, when creating a custom invoice, we have the option to set a specific date for the invoice. created_at is automatically generated.

Reply:
Sure I will update this , Actually i've checked this and the time for selected date is not updating while selecting the particular date , i have checked in the initial DB records it is present but now it is 00.00.00 that means it is not saving the time now , also in development it is same so need to fix that issue before time miss match . Till then we can put the testing of this issue on hold.